From 77cb4ad7606dc62999b1715039dc917e2505b679 Mon Sep 17 00:00:00 2001
From: BastianZim <10774221+BastianZim@users.noreply.github.com>
Date: Tue, 13 Apr 2021 11:33:42 +0200
Subject: [PATCH] Fix folder

---
 openml/config.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/openml/config.py b/openml/config.py
index 9e2e697d..1a9997b0 100644
--- a/openml/config.py
+++ b/openml/config.py
@@ -204,7 +204,7 @@ def _setup(config=None):
     # read config file, create directory for config file
     if not os.path.exists(config_dir):
         try:
-            os.mkdir(config_dir)
+            os.makedirs(config_dir, exist_ok=True)
             cache_exists = True
         except PermissionError:
             cache_exists = False
-- 
2.30.2

